app.py

```diff
@@ -164,13 +164,16 @@ def users_entries():
     prev_entries = get_entries(current_user.id)
     # adding tone aspect for each entry
     tones = []
-    print(prev_entries[0].timestamp)
-    if prev_entries is None:
-        flask.flash("Sorry, you have no entries at the moment, please add one.")
+    if len(prev_entries) == 0:
+        print("here")
+        flask.flash(
+            "Sorry, you have no entries at the moment, please add one at the bottom."
+        )
         return redirect(flask.url_for("home"))
     else:
         for entry in prev_entries:
-            tones.append(get_emotion(entry))
+            pass
+            # tones.append(get_emotion(entry))
```

Comment:
I commented it out, because it was causing problems with the app when you wanted to go to the entries page.